### PR TITLE
Create contact me section

### PR DESCRIPTION
**Pull request by Zeph180**
- Add contact me heading
- Add contact me text 
- Create a contact form
- Collect data with Formspree service
-

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -205,7 +205,20 @@ <h3 class="socials-header">LET'S CONNECT</h3>
...
+                <p>
+                    If you have an application you are interested in developing,
+                    a feature that you need built or a project that needs coding.
+                    I’d love to help with it 
+                </p>
+                <form action="https://formspree.io/f/mjvznweg" method="post">
+                    <input name="user-name" type="text" placeholder="Please enter your name" maxlength="30" required>
+                    <input name="user-email" type="email" placeholder="johndoe@gmail.com" required>
+                    <textarea name="user-msg" id="user-msg" cols="30" rows="10" maxlength="500" placeholder="write your message here" required></textarea>
+                    <button type="submit">Get in Touch</button>
```
Comment (Tufahel) [inline index.html]: Well done :+1: on implementing the form following every functionality from the requirement.
- [x] Kindly implement the `button interactions`,  such as: `hover`,  `enable`, etc. as per the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md).

Screenshot from 2022-12-06 19-29-59

File: index.css [hunk 2]
```diff
@@ -318,4 +326,71 @@ nav {
...
+    color: white;
+    margin-top: 70px;
+  }
+
+  .contact-sec h3 {
+    font-size: 40px;
+    font-weight: 700;
+    text-align: center;
+    margin-bottom: 0;
+  }
```
Comment (Tufahel) [inline index.css]: - [x] Kindly stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=qehlAKnut3MbM1FC-0). The position of `Contact Me` title and the `background image` is not as the given design. Following the design, your project will look more eye-catchy and professional.
- ***Your Design***
Screenshot from 2022-12-06 19-49-01

- ***Figma Design***
Screenshot from 2022-12-06 19-44-57

File: index.html [hunk 3]
```diff
@@ -205,7 +205,20 @@ <h3 class="socials-header">LET'S CONNECT</h3>
...
+                </form>
+            </div>
```
Comment (Tufahel) [inline index.html]: - [ ] ***[OPTIONAL]***
- Consider using the `section` tag for the `contact-sec` part. The reason is ***the section element provides more semantic syntax than the div element.*** Here is an [article](https://thisthat.dev/div-vs-section/), you can consider reading this. Thanks.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Zeph180 & @Tobby8629, :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Linters checks are passing
✔️ Use GitHub Flow
✔️ Meaningful commit messages.

## Required Changes ♻️

- [ ] Kindly remove the extra spaces from both sides of the `contact-me` section as they are not the same as the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=qehlAKnut3MbM1FC-0).

- ***Your Design***


_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 4]
```diff
@@ -318,4 +333,83 @@ nav {
...
+    color: #6070ff;
+    background-color: white;
+    font-size: 17px;
+    font-weight: 500;
+  }
+
+  .contact-sec button:hover {
+    background-color: #396df2;
+    color: white;
+  }
```
Comment (mukhammadrizooff) [inline index.css]: - [ ] [REQUIRED] Good job so far 👍🏼 I really appreciate what you have done 👍🏼, but I request you to fully comply with the request made by the [previous code reviewer](https://github.com/Zeph180/my-portfolio-microverse/pull/3#discussion_r1040978371) 🙏🏼  Below I will write down the correct `effect colors` for the `button`, please use them to `comply` with the request, I hope that your project will be approved in the next review 😃 : 
# 
```CSS
...

/* Enabled */
color: #6070FF;
background: #FFFFFF;

/* Hover */
color: #FFFFFF;
background: #6070FF;

/* Pressed */
color: #FFFFFF;
background: #2230D2;

/* Disabled */
color: #5E6C84;
background: #FFFFFF;

...
```

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Zeph180, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Zeph180, 

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️
- Good use of the Formspree ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**